example/benchmark/scripts/juliaimages/morphology/erode.jl

```diff
@@ -0,0 +1,12 @@
+using ImageMorphology
+using TestImages
+using BenchmarkTools, JSON3
+
+img = TestImages.shepp_logan(400)
+
+rst = @benchmark erode($img, 1) seconds=1
+
+Dict(
+    "memory" => rst.memory, # byte
+    "time" => median(rst.times)/1e6, # ms
+) |> JSON3.write
```

Comment:
Julia runner could choose to directly pass a `Dict`, but I want to make it language-agnostic so for consistency, I choose to let it pass a json String.